### DISCUSSION TITLE
REST Automation Question

### DISCUSSION THREAD

**Discussion by razerzhang** [Budibase/budibase#6793] [Help]
I want to use REST for Automation to send email. The configuration of Rest like blow pic.
![image](https://user-images.githubusercontent.com/76929114/179558017-bb549b0d-7086-4efd-bfe4-967e3c7bc8d2.png)
And Automation setting like that
![image](https://user-images.githubusercontent.com/76929114/179559666-9946dc1e-772a-45f5-8425-ec1006eea957.png)
database like that
![image](https://user-images.githubusercontent.com/76929114/179559801-05332785-62b8-4b89-a8c9-5f8676cee07c.png)
There are two question:
1.How I can use `{{ Current.User.Email }}` to customize body of the `REST` like the red line in picture one?
2.Why do this CONDITION Stopped? If there are some mistake in using Automation?
Thanks much

**Comment by R2bEEaton** [comment]
Hi!

### 1. How to add bindings to `REST` body.
https://docs.budibase.com/docs/rest-bindings

Go to the bindings tab and add a binding with a name "email" and whatever default value you want.
Now you can use the binding `{{ email }}` in the body section.

Importantly, go to your external data connector in your automation and set the "email" binding to the email you want from either the triggered row or some other thing. You can't use `{{ Current User.email }}` in Automations, you must pass `{{ Current User.email }}` into the automation by some other app action or use another way to get an email. Just as a rule of thumb `{{ Current User.* }}` is only available in app screens (edit: this may not be accurate).

### 2. Possible answer for the condition

Check your data's schema and make sure the `plan_budget` field is a number, not a string. That's all I can think of for that one, hope that helps.

**Reply by razerzhang** [reply]
Thank u so much.
But I found `{{ Current User.* }}` not only available in app screens, but also in Data, as you can see as blow pic.
![image](https://user-images.githubusercontent.com/76929114/179724374-5383d4e6-da70-49d9-853f-65171a392fac.png)
Through testing , find I can get `{{Cureent User.emal}}` ,but  when write `SQL query` and insert  `{{Cureent User.emal}}` is not available. I don`t known the reason.

**Reply by R2bEEaton** [reply]
Interesting, that's new to me.
